examples/decision_transformer/lamb.py

```diff
@@ -0,0 +1,208 @@
+""" PyTorch Lamb optimizer w/ behaviour similar to NVIDIA FusedLamb
```

Comment:
Add meta headers and add a note saying that this is copied + url from which it's taken.